rotkehlchen/tests/db/test_db.py

```diff
@@ -1370,3 +1375,83 @@ def test_values_are_present_in_db(database, enum_class, table_name):
     for enum_class_entry in enum_class:
         r = cursor.execute(query, (enum_class_entry.value,))
         assert r.fetchone() == (1,)
+
+
+def test_known_locations(user_data_dir, rotkehlchen_api_server):
```

Comment:
This has no place in the db tests. The DB tests are UNIT tests. You are initializing an entire api server here. This should be in the api tests.